github/pulls.go

```diff
@@ -134,6 +135,32 @@ func (s *PullRequestsService) Get(owner string, repo string, number int) (*PullR
 	return pull, resp, err
 }
 
+// GetRaw gets raw (diff or patch) format of a pull request.
+func (s *PullRequestsService) GetRaw(owner string, repo string, number int, opt RawOptions) ([]byte, *Response, error) {
```

Comment:
What was your reason to go with returning `[]byte` instead of `string`?

I've thought about it some more, and `string` seems to be a better fit to me. Both diff and patch are textual formats. go-github already has a few endpoints that return text, such as [`Markdown`](https://godoc.org/github.com/google/go-github/github#Client.Markdown), [`Octocat`](https://godoc.org/github.com/google/go-github/github#Client.Octocat) and [`Zen`](https://godoc.org/github.com/google/go-github/github#Client.Zen).

Do you have arguments for `[]byte`?